client/tso_client.go

```diff
@@ -259,6 +267,7 @@ func (c *tsoClient) updateTSOGlobalServURL(url string) error {
 	log.Info("[tso] switch dc tso global allocator serving url",
 		zap.String("dc-location", globalDCLocation),
 		zap.String("new-url", url))
+	c.scheduleUpdateTSOConnectionCtxs()
 	c.scheduleCheckTSODispatcher()
```

Comment:
Does `updateTSOLocalServURLs` need to run `scheduleCheckTSODispatcher`?

Reply:
Yes, and it has already done this check: https://github.com/JmPotato/pd/blob/50c80407cd57e96a85452dd1601fcb41c1f263cf/client/tso_client.go#L251